### ISSUE TITLE
QA/ADE Finding: Focus is not set to the top of the page for the Submission History tab

### ISSUE THREAD

**Issue by zslocum1** [department-of-veterans-affairs/va.gov-team#138051]
## Background/Context

Focus is automatically set to the results summary on page load, causing screen reader users to miss all of the previous content. We shouldn't set focus on page load and allow the user to navigate through all of the content using their preferred method. Focus should be set on the summary after subsequent searches.

JAWS: On page load, focus immediately moves to "Showing 1-20 of...." instead of the top of the page or the "Submission history" heading

## Affected Devices

- Screen reader
- JAWS
- Talkback

## Resources

- [Accessibility defect severity rubric
](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric)
## Expected vs. Actual
**Expected**: 
When navigating to the Submission History tab it is expected that focus is managed according to the user, so they can choose to navigate through all of the content based on their preferred method.

**Actual**: 
Instead, upon the Submission history page opening, focus immediately moves to "Showing 1-20of".

## Examples

JAWS example:

https://github.com/user-attachments/assets/8b569963-7c93-4d95-a1f4-4a3f1585ab0f

**Comment by darbyemadewell** [comment]
Blocked on this ticket until we can see POA locally (I can't get to the Submissions History page to test a fix). Accreditation is going to take a look soon. CC: @zslocum1

**Comment by ataker** [comment]
Verified fixed
